aws/internal/service/rds/filter/filter.go

```diff
@@ -0,0 +1,30 @@
+package filter
```

Comment:
Since there are multiple services that use this pattern, would it make sense to implement the generator pattern similar to `keyvaluetags`? e.g. `namevaluesfilters`

Then it might be a similar interface such as:

```
type NameValuesFilters map[string][]string

func New(m map[string][]string) NameValuesFilters

func (NameValuesFilters) AutoscalingFilters() []*autoscaling.Filter
func (NameValuesFilters) DatabasemigrationserviceFilters() []*databasemigrationservice.Filter
func (NameValuesFilters) DocdbFilters() []*docdb.Filter
func (NameValuesFilters) Ec2Filters() []*ec2.Filter
func (NameValuesFilters) FsxFilters() []*fsx.Filter
func (NameValuesFilters) ImagebuilderFilters() []*imagebuilder.Filter
func (NameValuesFilters) LicensemanagerFilters() []*licensemanager.Filter
func (NameValuesFilters) NeptuneFilters() []*neptune.Filter
func (NameValuesFilters) RdsFilters() []*rds.Filter
func (NameValuesFilters) Route53resolverFilters() []*route53resolver.Filter
```

Reply:
yes, it probably would.